econml/tests/test_deepiv.py

```diff
@@ -93,6 +93,7 @@ def test_deepiv_shape(self):
             deepIv.fit(Y=y, T=t, X=x, Z=z)
             # do something with predictions...
             deepIv.predict(T=t, X=x)
+            deepIv.effect(x, np.zeros_like(t), t)
```

Comment:
Can we also add a test with no treatments passed in?

Reply:
@moprescu Not yet, because apparently I forgot to enable deep IV to take vectors for treatments and responses...

Reply:
@moprescu Now addressed by #59